charts/polkadot-watcher/templates/alertrules.yaml

```diff
@@ -61,6 +61,13 @@ spec:
       for: 2m
       labels:
         severity: info
-        origin: {{ .Values.prometheusRules.origin }}  
-            
+        origin: {{ .Values.prometheusRules.origin }}
+    - alert: ValidatorRewardDestinationChanged
+      annotations:
+        message: 'Target {{`{{ $labels.name }}`}} has changed his reward destination'
```

Comment:
would it be possible and/or make sense to include the new destination?

Reply:
I agree. It has been addressed here: https://github.com/w3f/polkadot-watcher/pull/91

the final outcome is that the message will route you [here](https://polkadot.subscan.io/validator/11uMPbeaEDJhUxzU4ZfWW9VQEsryP9XqFcNRfPdYda6aFWJ) for example